### DISCUSSION TITLE
clang有没有什么调试工具

### DISCUSSION THREAD

**Discussion by TommyPlayer-c** [arcsysu/SYsU-lang#35] [Q&A] [answered]
做实验时调试只会 printf，想问一下 clang 有没有类似 gdb 的工具，这样 printf 每次都得重新编译一遍，效率太低了

**Comment by wu-kan** [comment, marked as answer]
可以试试 `lldb`？

**Reply by TommyPlayer-c** [reply]
助教能不能提供简明教程，比如给出调试执行的具体命令，就像在 readme 给出的命令那样

**Reply by wu-kan** [reply]
`lldb` 应该不是完成这个实验的必选项？想学习它的用法，网上应该也有很多中文材料了？

**Reply by Yorkking** [reply]
> `lldb` 应该不是完成这个实验的必选项？想学习它的用法，网上应该也有很多中文材料了？

换句话说，TA也不会🤣😂

**Reply by wu-kan** [reply]
> > `lldb` 应该不是完成这个实验的必选项？想学习它的用法，网上应该也有很多中文材料了？
> 
> 换句话说，TA也不会🤣😂

faQ♂
